src/Bicep.Core/Navigation/SyntaxBaseExtensions.cs

```diff
@@ -75,42 +74,6 @@ public static string ToText(this SyntaxBase syntax, string indent = "", string?
             document.Layout(sb, indent, newLineSequence ?? Environment.NewLine);
             return sb.ToString();
         }
```

Comment:
Is there a reason ToText is not directly on SyntaxBase like ToString()?  It's going to be quite confusing having both of these and not know what the difference is, especially if they're in different places.

Reply:
The goal here is to remove all `ToText()` instances with `ToString()` or calls to the v2 formatter if necessary (to be addressed in a subsequent PR).